backend/config/curation-api.yml

```diff
@@ -164,6 +165,7 @@ paths:
         content:
           application/json:
             schema:
+              additionalProperties: false
```

Comment:
No, it is not. I remove it in this PR so that I can use this portion of schema on the response for this endpoint. Otherwise, `additionalProperties: false` conflicts with the addition of `publisher_metadata`, which is now being returned.